test/func/conftest.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) 2014 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pytest
+import swift_scality_backend.policy_configuration as policy_configuration
+
+import os_utils
+import sproxyd_utils
+
+
+def pytest_addoption(parser):
+    parser.addoption("--host-ip", action="store")
+    parser.addoption("--allow-encoded-slashes", action="store")
+    parser.addoption("--os-auth-url", action="store")
+    parser.addoption("--os-identity-api-version", action="store")
+    parser.addoption("--os-demo-username", action="store")
+    parser.addoption("--os-demo-password", action="store")
+    parser.addoption("--os-demo-tenantname", action="store")
+    parser.addoption("--os-admin-username", action="store")
+    parser.addoption("--os-admin-password", action="store")
+    parser.addoption("--os-admin-tenantname", action="store")
+    parser.addoption("--sproxyd-numbers", action="store")
+
+
+def get_demo_swift_connection(request):
+    # FIXME : read user, password, etc... from somewhere ?
```

Comment:
I don't know if pytest used argparse underneath, I suspect is does, if so I used this trick http://stackoverflow.com/questions/10551117/setting-options-from-environment-variables-when-using-argparse in the past with great success.